components/kucoin_futures/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/kucoin_futures",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Pipedream Kucoin Futures Components",
   "main": "dist/app/kucoin_futures.app.mjs",
```

I think this line should now be just
```suggestion
  "main": "kucoin_futures.app.mjs",
```
since you are working with just plain javascript